tests/sweep_runner/test_param_sweep_runner.py

```diff
@@ -346,6 +421,11 @@ def sweep_method(data: np.ndarray, param_1: int):  # type: ignore
         "httomo.method_wrappers.generic.import_module", return_value=FakeModule
     )
 
+    mocker.patch(
+        "httomo.sweep_runner.param_sweep_runner._preview_modifier",
+        return_value=[50, 50],
+    )
+
```

Comment:
Similar to my comments about asserting on private members of an object, I'm not overfly fond of patching private methods/functions, because they should be an implementation detail that the test shouldn't need to care about.

In this test, it's not doing paganin kernel estimation stuff to modify the preview, so would it be possible to instead patch the entire public function `ParamSweepRunner.modify_loader_preview` to leave the preview alone, rather than patching the private function `_preview_modifier`?

Reply:
Come to think of it, I feel like maybe not patching it at all would be an even better strategy.

We'd want to make sure that in the absence of paganin, the newly added feature _doesn't modify the preview at all_. If the preview _did_ get changed by the new logic for a sweep pipeline that _doesn't_ have paganin, we'd want to know about it, and mocking any of the logic for modifying the preview would mask that error.

So, I'd say to remove the patching completely, we want to know if the new feature is messing with the preview when paganin isn't present, and not patching the new logic would tell us that.

Reply:
yep, got rid of it